### ISSUE TITLE
Configure forking context with conf file

### ISSUE THREAD

**Issue by labianchin** [spotify/flo#103]
Currently forking can be disabled with environment variable `FLO_FORKING=false`.

I would be great to have this configuration in typesafe config, like defining something in [`reference.conf`](https://github.com/spotify/flo/blob/master/flo-runner/src/main/resources/reference.conf):

```
flo.forkingEnabled=true
flo.forkingEnalbed=${?FLO_FORKING_ENABLED}
```

**Comment by rouzwawi** [comment]
How about also disabling it by default? Seems to be the cause of a number of bugs lately: #100 #101 #102.

Object serialization and JVM forking is tricky, so we should let `ForkingContext` accumulate some mileage before being enabled by default.

**Comment by rouzwawi** [comment]
Merged #104 now. Seems it missed `0.2.11` though.

**Comment by danielnorberg** [comment]
This has been released. Closing.
